scripts/fast-reboot

```diff
@@ -407,12 +407,13 @@ docker kill lldp > /dev/null
 systemctl stop lldp
 
 if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    # Kill teamd processes inside of teamd container with SIGINT to allow them to send last LACP frames
     # Kill teamd, otherwise it gets down all LAGs
     # We call `docker kill teamd` to ensure the container stops as quickly as possible,
     # then immediately call `systemctl stop teamd` to prevent the service from
     # restarting the container automatically.
     # Note: teamd must be killed before syncd, because it will send the last packet through CPU port
-    # TODO: stop teamd gracefully to allow teamd to send last valid update to be sure we'll have 90 seconds reboot time
+    docker exec -i teamd pkill -INT teamd
```

Comment:
Might be safer to do something like https://github.com/Azure/sonic-utilities/blob/15e24b0e9b7e9ef30b667219cd40dfb8c0dd60c8/scripts/fast-reboot#L399 in case teamd has quit already.